.gitignore

```diff
@@ -8,3 +8,4 @@
 /dist/
 .pipfile
 .tox
+.idea/
```

Comment:
We prefer to not add editor-specific ignores to project `.gitignore`s. Please can you remove this and instead set this in a global `.gitignore` on you computer?